metabrainz/static/css/main.less

```diff
@@ -211,6 +211,10 @@ h1.page-title, h2.page-title {
   }
 }
 
+// Projects page
+#projects {
+  img { width: 90px; }
```

Comment:
Are we sure that we want absolutely all images on this page to be this wide? I think it'd probably be better to add a `logo` class or something like that, and use that class for the images. (And apply this `width` to the class instead of the `<img>` element in general.)